CHANGELOG.md

```diff
@@ -23,8 +23,9 @@
 
 ### Internals
 * Android: build with NDK r22. Make `-Wl,-gc-sections` an interface linker flag, which reduces code size because Core is compiled `-fdata-sections` and `-ffunction-sections`. Use `-Oz` even when we enable link-time optimization (previously we used with `-O2`). ([#4407](https://github.com/realm/realm-core/pull/4407))
+* The DB class now supports opening a realm file on a write-only file system.
```

Comment:
This should have some user-visible effects too, right? I think it enables ThreadSafeReferences on Immutable Realms?